Argo/Operators/Operators.swift

```diff
@@ -8,44 +6,72 @@ infix operator <||? { associativity left precedence 150 }
 // MARK: Values
 
 // Pull value from JSON
-public func <| <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) -> Decoded<A> {
-  return decodedJSON(json, forKey: key) >>- A.decode
+public func <| <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) throws -> A {
+  return try A.decode(decodedJSON(json, forKey: key))
 }
 
 // Pull optional value from JSON
-public func <|? <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) -> Decoded<A?> {
-  return .optional(json <| key)
+public func <|? <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) throws -> A? {
+  do {
+    return try .Some(json <| key)
+  } catch let error as DecodedError {
+    switch error {
+    case .MissingKey(_): return .None
```

Comment:
this is now the way we need to check to see if a missing key error is "OK" for the case where the user says it may or may not be there